src/App.jsx

```diff
@@ -13,34 +15,103 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const RESET = '';
+const SORT_ALPHABETICALLY = 'alphabet';
+const SORT_BY_LENGTH = 'length';
```

Comment:
*optional. Since `enum` isn't required

```suggestion
enum SortBy {
  ALPHABETICALLY = 'alphabet',
  LENGTH = 'length'
  DEFAULT = '',
}
```